sale_view_adj_sst/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+Added Filter on Sales Management Searchview
+- "Order Line" - search that contains search word from order line.
```

Comment:
```suggestion

* Adjusts the sales order search view.
```

We can keep it as simple as this here, to avoid adding detailed explanations every time we update the module.